composer/models/huggingface.py

```diff
@@ -170,6 +170,9 @@ def load_huggingface_tokenizer_from_saved_state(
                             for line in saved_content['content']:
                                 _tmp_file.write(line)
                                 _tmp_file.write('\n')
+                    elif saved_content['file_extension'] == '.py':
```

Comment:
This must have gotten accidentally deleted at some point, because I originally fixed in https://github.com/mosaicml/composer/pull/2279/files

Reply:
ah makes sense, it must have still worked for some reason w/ transformers 4.30.2 but broken w/ transformers 4.31

Reply:
Yeah, if you read the description on that pr, it only affected an older version of transformers as far as i can tell, but maybe they changed it back to the way it was before? not exactly sure, but either way, what you've added is the right fix.